plugins/ctf/lttng-live/viewer-connection.c

```diff
@@ -55,16 +52,16 @@ static ssize_t lttng_live_recv(struct bt_live_viewer_connection *viewer_connecti
 	size_t copied = 0, to_copy = len;
 	struct lttng_live_component *lttng_live =
 			viewer_connection->lttng_live;
-	int fd = viewer_connection->control_sock;
+	BT_SOCKET sock = viewer_connection->control_sock;
 
 	do {
-		ret = recv(fd, buf + copied, to_copy, 0);
+		ret = recv(sock, buf + copied, to_copy, 0);
```

Comment:
Ideally, `recv()` and `send()` would also be wrapped to make the socket library consistent. Not a priority at the moment, though.